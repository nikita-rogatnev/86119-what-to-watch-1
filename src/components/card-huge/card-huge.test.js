import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import CardHuge from "./card-huge";

Enzyme.configure({adapter: new Adapter()});

it(`CardHuge renders correctly`, () => {
  const tree = shallow(<CardHuge/>);

  expect(toJson(tree)).toMatchSnapshot();
});
