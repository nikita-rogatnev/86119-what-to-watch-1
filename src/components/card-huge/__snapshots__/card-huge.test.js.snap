// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHuge renders correctly 1`] = `
<section
  className="movie-card"
>
  <div
    className="movie-card__bg"
  >
    <img
      alt="The Grand Budapest Hotel"
      src="img/bg-the-grand-budapest-hotel.jpg"
    />
  </div>
  <h1
    className="visually-hidden"
  >
    WTW
  </h1>
  <ConnectFunction />
  <div
    className="movie-card__wrap"
  >
    <div
      className="movie-card__info"
    >
      <div
        className="movie-card__poster"
      >
        <img
          alt="The Grand Budapest Hotel poster"
          height="327"
          src="img/the-grand-budapest-hotel-poster.jpg"
          width="218"
        />
      </div>
      <div
        className="movie-card__desc"
      >
        <h2
          className="movie-card__title"
        >
          The Grand Budapest Hotel
        </h2>
        <p
          className="movie-card__meta"
        >
          <span
            className="movie-card__genre"
          >
            Drama
          </span>
          <span
            className="movie-card__year"
          >
            2014
          </span>
        </p>
        <div
          className="movie-card__buttons"
        >
          <button
            className="btn btn--play movie-card__button"
            type="button"
          >
            <svg
              height="19"
              viewBox="0 0 19 19"
              width="19"
            >
              <use
                xlinkHref="#play-s"
              />
            </svg>
            <span>
              Play
            </span>
          </button>
          <Link
            className="btn btn--list movie-card__button"
            to="/mylist"
          >
            <svg
              height="14"
              viewBox="0 0 18 14"
              width="18"
            >
              <use
                xlinkHref="#in-list"
              />
            </svg>
            <span>
              My list
            </span>
          </Link>
        </div>
      </div>
    </div>
  </div>
</section>
`;
